spec/treasury_and_analytics.md

```diff
@@ -0,0 +1,135 @@
+# Treasury and Analytics
+
+This specification provides details of BitDAO treasury and analytics pages, as well as associated
+required services.
+
+We first describe how services are constructed, then we describe how the data is presented to users.
+
+The treasury and analytics approach taken here is based on daily values. This specification is not
+intended to serve shorter periods or live updates (the backend services will need to be re-evaluated
+for performance and flexibility, and likely requires rebuilding).
+
+## Features
+
+- Idempotent pipeline
+- Static page generation
+- Serverless using object storage, i.e., no key-value stores or relational database administration
+- Decentralized analytics by enabling self-contained pages
+
+## Analytics
+
+Data retrieval:
+
+- A serverless code executable retrieves token quantities and spot prices once a day.
+- Retrieved values are saved to S3 (as CSV or JSON) using the following filename convention:
+    - {token}-{yyyy}-{mm}-{dd}.json- If spot prices are pulled from multiple sources, use the
+      following filename convention:
```

Comment:
question: is part of the sentence missing? which convention should be used when pulling data from multiple sources?

Reply:
yes, thx. formatting too.. will fix